### ISSUE TITLE
Error exporting to Visual Studio...

### ISSUE THREAD

**Issue by smattiso** [MicrosoftDocs/mixed-reality#332]
I'm using a Surface Pro 4 to build my app, and it will do "Remote to Device" holographic emulation just fine, however when I try to Build (following the instructions under 'Export to the Visual Studio solution'), I get the error "Could not start compilationException: [sic] 'C:\Program Files (x86)\MSBuild\14.0\Bin\csc.exe' not found, either .NET 4.5 is not installed or your OS is not Windows 8/8.1."

Running the .NET installer, I get ".NET Framework 4.7.1 or a later update is already installed on this computer." so I'm certain I've got at least 4.5 installed.
Running "wmic product get description | findstr /C:".NET Framework" shows the 4.6.1 SDK among other packages.

Do I really need to downgrade from Windows 10 to an inferior operating system in order to develop for the cutting edge HoloLens?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8969b329-9061-f1c0-70c7-b99aa0aa1b94
* Version Independent ID: a80974ef-fbc2-7559-7711-b899e349c010
* Content: [Holograms 100 - Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/holograms-100#)
* Content Source: [mixed-reality-docs/holograms-100.md](https://github.com/MicrosoftDocs/mixed-reality/blob/master/mixed-reality-docs/holograms-100.md)
* Service: **unspecified**
* GitHub Login: @keveleigh
* Microsoft Alias: **kurtie**

**Comment by smattiso** [comment]
It looks like other people have been told to go to forums.hololens.com ... Meanwhile, I feel like I've been waiting on hold for two days. That's customer service for you!

**Comment by smattiso** [comment]
Judging by certain accepted solutions on those forums, it appears there's a Windows 8/8.1 SDK and a Windows 10 SDK.
I can't be sure that this is my issue until I've resolved the conflict and actually compile something, but it feels like it might be related...

**Comment by keveleigh** [comment]
Which version of Visual Studio do you have installed?
Which SDK version do you have installed?

**Comment by smattiso** [comment]
VS 2017. I'm uninstalling all the components and then reinstalling UWP SDK 10.0.16299.0 now.
Checking only UWP Dev and Game Dev with Unity.
...
Same error.

**Comment by smattiso** [comment]
Under Program Files:
Windows Kits has both 10 and 8.1 directories. Is that normal?
Under MSBuild, there's a 14.0 and a 15.0, but neither one has a Bin.

**Comment by keveleigh** [comment]
I have both 10 and 8.1, though 8.1 is mostly empty.
I also have a 14.0 and a 15.0 under MSBuild, neither with a Bin.

Which Unity version are you using?

**Comment by smattiso** [comment]
Unity 5.5.0f3

**Comment by keveleigh** [comment]
Ahh, I believe there's the issue. If I remember correctly, older versions of Unity weren't compatible with VS2017 and were hardcoded to look specifically for certain VS2015 paths, which sounds like exactly what you're running into.

Our [current Unity recommendation](https://docs.microsoft.com/en-us/windows/mixed-reality/install-the-tools) is Unity 2017.4.

I've recently used Unity 5.6.5 and newer without issue.

**Comment by smattiso** [comment]
Ahh, I see. Thanks. I'll download the update right away!

**Comment by smattiso** [comment]
Looks like it's building correctly now. Thanks again for the help!
